config/_shared.json

```
@@ -149,7 +149,7 @@
 	],
 	"restricted_me_access": true,
 	"theme_color": "#055d9c",
-	"reskinned_flows": [ "onboarding", "launch-site" ],
+	"reskinned_flows": [ "onboarding", "launch-site", "onboarding-with-email" ],
```

That's minor but maybe we could keep those values ordered:
```suggestion
	"reskinned_flows": [ "launch-site", "onboarding", "onboarding-with-email" ],
```